website/docs/sig-suggestion.md

```diff
@@ -0,0 +1,101 @@
+---
+id: sig-suggestion
+title: Automatically suggesting method signatures
+sidebar_label: Suggesting sigs
+---
+
+In general, Sorbet does not attempt to infer parameter and return types for
+method methods without [method signatures](sigs.md).
+
+(For further context, see
+[Why does Sorbet sometimes need type annotations?](why-type-annotations.md).)
+
+However, Sorbet has best effort support for **suggesting** method signatures
+that the user may want to insert into the codebase. By best effort we mean that
+sometimes Sorbet will fail to infer a any useful types. Despite limitations,
+Sorbet is able to suggest signatures in many cases. There are two main ways to
+get Sorbet to suggest a signature for a method.
+
+## Quickfix code actions in `# typed: strict` or higher
+
+It's an error in a [`# typed: strict`](static.md) file to have a method without
+a signature. When reporting these errors, Sorbet runs its signature suggestion
+logic to attempt to provide an autocorrect.
+
+In editors, these [autocorrects] are surfaced as **quickfix code actions**.
+Different language clients have different ways to apply these code actions, but
+in VS Code they are surfaced either by clicking on the 💡 lightbulb icon, or by
+pressing `Ctrl` + `.` (or `Cmd` + `.`, depending on your platform).
```

Comment:
No emoji for `Cmd`? 😛

Reply:
Yeah I was mixing emoji and non-emoji :shame:

another doc in the PR uses ⌘

I'm always unsure what to do here, because idk if windows people will understand that ⌃ means control, and it looks weird to mix `Ctrl` and ⌘